### PR TITLE
add page for info on translating sports landing page

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by tanyaparker** [comment]
Videos secret until Hour of Code

File: pegasus/sites.v3/code.org/public/translate/sports.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+**Both tutorials**
+
+1. [code.org/sports landing page](https://crowdin.com/translate/codeorg/56/enus-es#q=sports)
+1. [code.org/sports videos](https://crowdin.com/translate/codeorg/56/enus-es#q=video_)
+
+## Translating the video captions
+
+1. To use Amara, first click on a video link below.
+1. On the left, you'll see a list of languages. If you don't see your language listed, click "Add a new language". A language with a yellow dot means it's been started but incomplete; please go ahead and complete the translation. A language with a green dot means it's been completed and published.
+1. When you begin you'll see the English track on the left and you can enter your translations on the right. Feel free to edit the timing if necessary. When complete, click the Publish button in the top right.
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/translate/sports.md]: "at" the top right maybe?

Comment (breville) [reply pegasus/sites.v3/code.org/public/translate/sports.md]: Or "in the top right corner"?

**Comment by breville** [comment]
lgtm

**Pull request merged**